lib/backend/k8s/resources/globalbgpconfig.go

```diff
@@ -80,17 +80,19 @@ func (c GlobalBgpConfigConverter) FromKVPair(kvp *model.KVPair) (CustomK8sResour
 	if err != nil {
 		return nil, err
 	}
-	tpr := thirdparty.GlobalBgpConfig{
-		Metadata: metav1.ObjectMeta{
-			Name: name,
+	crd := custom.GlobalBgpConfig{
+		TypeMeta: metav1.TypeMeta{
```

Comment:
I don't fully understand why we needed to add TypeMeta for GlobalBgpConfig but not for other resources.  This needs more investigation.